lib/godmin/authorization.rb

```diff
@@ -22,7 +22,7 @@ def authorize(record, query = nil)
     end
 
     def policy(record)
-      policies[record] ||= PolicyFinder.find(record, engine_namespace).new(admin_user, record)
+      policies[record] ||= PolicyFinder.find(record, engine_wrapper.namespace.to_s.underscore).new(admin_user, record)
```

Comment:
Can we pass the `engine_wrapper.namespace` instead and change the PolicyFinder to handle that? Might break some backward compatibility though...